src/runnerComponent.js

```diff
@@ -114,6 +114,17 @@ export default function runnerComponentFactory(container = null, config = {}, te
          */
         getRunner() {
             return runner;
+        },
+
+        /**
+         * Set loaded plugins
+         * Should be called before render
+         * @param {Array|Object} loadedPlugins
+         */
+        setLoadedPlugins(loadedPlugins) {
+            if (!plugins.length) {
+                plugins = loadedPlugins;
```

Comment:
But if `loadedPlugins` is an object (as specified in the JSDoc), `plugins.length` will always be falsy.